### DISCUSSION TITLE
filed: { not: true } return null result

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#9087] [Q&A]
Hi, i have a row like this

id      |     success(boolen here)  | user
1       |        null                           | 2

then I query
```ts
const getTheRow= await prm.table.findFirst({
        where: {
            success: { not: true },
            user: 2,
        },
    })
```

But I get return null, why?? but if I query `success:null` it works. I don't understand. Must I set `success=false` when I created the row?

**Comment by ryands17** [comment]
@IRediTOTO 👋 

Using `{ not: true }` will return false and null values both as it satisfies the query. If you don't want null values you need to combine it like this or set `success: false`.

```ts
const getTheRow= await prm.table.findFirst({
        where: {
            success: { not: true },
            success: { not: null },
            user: 2,
        },
})
```

**Reply by IRediTOTO** [reply]
I have no idea for this, i must set `success:false ` when create row. But fine, thank you for reply me :D

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
